### PR TITLE
Update Columns mobile styling for tables

### PR DESCRIPTION
**Pull request by rrusher**
Added styling for mobile view under 600px.

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1680

Test URLs:

Before: https://main--exlm--adobe-experience-league.hlx.page/en/articles/analytics/analytics-article
After: https://exlm-1545--exlm--rrusher.hlx.page/en/articles/analytics/analytics-article

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -20,23 +20,92 @@
...
+    display: flex;
+  }
+
+  .columns.block.b-alt-row .header-label {
+    display: none;
+  }
+
+  .columns.block.b-alt-row > div > div {
+  border: none;
+  }
```
Comment (noormd08) [inline blocks/columns/columns.css:91]: Looks like these class .columns.block.b-alt-row > div > div are repeated. Can we merge all the css properties and keep it in single.

After this, we can keep first-child and last-child below to it.

File: blocks/columns/columns.js [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+        const cellContent = cell.querySelector('.cell-content');
+        if (headerLabel && cellContent) {
+          cell.textContent = cellContent.textContent;
+        }
+      });
+    });
+  }
+}
+
+window.addEventListener('resize', updateHeaders);
```
Comment (noormd08) [inline blocks/columns/columns.js:37]: Seems fine, but good to go with resizeObserver to keep track of changes to the size of an element.

File: blocks/columns/columns.css (hunk 1 above)
Comment (rrusher) [reply blocks/columns/columns.css]: Updated.

File: blocks/columns/columns.js (hunk 2 above)
Comment (rrusher) [reply blocks/columns/columns.js:37]: Since the use case is only when someone is resizing the window, I opted to keep it simple.

**Review by noormd08 — APPROVED** [review]

**Review by noormd08 — APPROVED** [review]
Approved and merging for now. 
As mentioned earlier, please raise the PR from EXLM Project feature branch to main branch

**Pull request merged**